Update HelloWorld.java

mansip29 opened the pull request:
hey checkout this!!!

mansiprajapati5 committed:
Update HelloWorld.java

hey checkout this!!!

mansip29 committed:
Program for prime numbers

mansip29 committed:
Program for online menu card of the hotels.

yogeshprajapati1985 commented on Menu/src/Option.java:
```diff
@@ -0,0 +1,105 @@
+		case 7:
+			System.out.println("Thank you for visiting");
+			break;
+			
```
You can add default case here.

yogeshprajapati1985 commented on Menu/src/Option.java:
```diff
@@ -0,0 +1,105 @@
+import java.util.Scanner;
```
It is best practice to have documentation comment.

yogeshprajapati1985 replied:
Please refer below for documentation comments:

https://www.tutorialspoint.com/java/java_documentation.htm

mansip29 replied:
okay.
Thank you!!

yogeshprajapati1985 commented on Menu/src/Option.java:
```diff
@@ -0,0 +1,105 @@
+	System.out.println("| 2. Coca cola |");
+	System.out.println("| 3. Lemonade |");
+	System.out.println("| 4. Nectar |");
+}
```
It is best practice to have documentation comment. Please do it for each method.

yogeshprajapati1985 commented on prime/src/prime/demo.java:
```diff
@@ -0,0 +1,36 @@
+package prime;
+import java.util.Scanner;
+class First{
+	
```
It's best practice to have appropriate method name.

mansiprajapati5 committed:
added default case

added default case and comments as well!!

mansiprajapati5 committed:
changed the name of method

changed the name of the method from get() to check().

mansip29 committed:
Login page using swing

check the code. My eclipse is now supporting swt & awt , so I have not run the code yet.

mansiprajapati5 committed:
loginPage

mansiprajapati5 committed:
deleted the old one

mansip29 committed:
loginPage

mansiprajapati5 committed:
Added the documentation comment.

mansip29 committed:
text to speech conversation program in java
